$OpenBSD: patch-scripts_Install,v 1.1.1.1 2010/12/22 18:38:03 jasper Exp $

Don't install these directories, it's taken care of already
by mtree, or will be with @sample in the PLIST.

--- scripts/Install.orig	Tue Dec 21 16:02:05 2010
+++ scripts/Install	Tue Dec 21 16:02:29 2010
@@ -18,10 +18,6 @@ p=${prefix}
 e=${etcdir}
 v=${vardir}
 
-install -d ${destdir}${prefix}/bin
-install -d ${destdir}${prefix}/etc
-install -d ${destdir}${vardir}/log/yaws
-
 cat yaws.template | \
     ./Subst %yawsdir% ${prefix}/lib/yaws | \
     ./Subst %vardir% ${vardir} | \
